R/pkg/R/functions.R

```diff
@@ -1105,6 +1105,20 @@ setMethod("monthname",
             column(jc)
           })
 
+#' @details
+#' \code{dayname}: Extracts the three-letter abbreviated day name from a
```

Comment:
I'm fine with the function name DAYNAME, but shall we make it clearer in the doc? `day-of-week name` instead of `day name`. This is also what snowflake does: https://docs.snowflake.com/en/sql-reference/functions/dayname